src/File.php

```diff
@@ -981,28 +981,12 @@ public static function format_size($size)
      * Convert a php.ini value (eg: 512M) to bytes
      *
      * @param  string $iniValue
-     * @return int
+     * @return float
      */
     public static function ini2bytes($iniValue)
     {
-        $iniValues = str_split(trim($iniValue));
-        $unit = strtolower(array_pop($iniValues));
-        $quantity = (int) implode($iniValues);
-        switch ($unit) {
-            case 'g':
-                $quantity *= 1024;
-                // deliberate no break
-            case 'm':
-                $quantity *= 1024;
-                // deliberate no break
-            case 'k':
-                $quantity *= 1024;
-                // deliberate no break
-            default:
-                // no-op: pre-existing behaviour
-                break;
-        }
-        return $quantity;
+        Deprecation::notice('5.0', 'Use Convert::memstring2bytes instead');
```

Comment:
Ah, I think you're right sorry. We also don't set a per-module deprecation in the same way we do for framework.